Controls/Grid/Grid.UWP/Model/GridModel.Columns.cs

```diff
@@ -90,6 +93,37 @@ internal static void UpdateFilterMemberAccess(IFieldInfoData data, IEnumerable<F
             }
         }
 
+        internal static PropertyInfoFieldInfo InitializePropertyInfo(string nestedPropertyPath, Type type)
+        {
+            string[] splitNestedProperty = nestedPropertyPath.Split(new char[] { NestedPropertySeparator });
+            PropertyInfoFieldInfo nestedPropertyInfo = null;
+            List<PropertyInfo> propertyInfos = new List<PropertyInfo>();
+            if (splitNestedProperty[0] == type.Name)
+            {
+                PropertyInfo propertyInfo = null;
+                for (int i = 1; i < splitNestedProperty.Length; i++)
+                {
+                    string propertyName = splitNestedProperty[i];
+                    propertyInfo = type.GetProperty(propertyName);
+                    if (propertyInfo == null)
+                    {
+                        return null;
+                    }
+
+                    propertyInfos.Add(propertyInfo);
+                    type = propertyInfo.PropertyType;
+                }
+
+                var propertyAccess = Telerik.Data.Core.BindingExpressionHelper.CreateGetNestedValueFunc(propertyInfos, splitNestedProperty[0]);
+                var propertySetter = Telerik.Data.Core.BindingExpressionHelper.CreateSetNestedValueFunc(propertyInfos, splitNestedProperty[0]);
```

Comment:
@APopatanasov I meant here to check if property is writable and avoid calling CreateSetNestedValueFunc :)

Reply:
@hshristov  Sorry for the misunderstanding. Here is the PR with the right check: https://github.com/telerik/UI-For-UWP/pull/303